### PR TITLE
Add a make target for Jenkins

### PR DESCRIPTION
**Pull request by ray-milkey**

### CODE REVIEW

**Review by ray-milkey — COMMENTED** [review]
retest recheck

**Comment by zdw** [comment]
test this please

File: Makefile [hunk 1]
```diff
@@ -25,10 +29,22 @@ deps: # @HELP ensure that the required dependencies are in place
...
+	@if [ ! -d "../build-tools" ]; then cd .. && git clone https://github.com/onosproject/build-tools.git; fi
+
+golang-ci: # @HELP install golang-ci if not present
+	go env GOPATH
+	echo ${PATH}
+	golangci-lint --version || curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b `go env GOPATH`/bin v1.23.7
+	ls -l `go env GOPATH`/bin
+
+jenkins-tools: # @HELP installs tooling needed for Jenkins
+	cd .. && go get -u github.com/jstemmer/go-junit-report && go get github.com/t-yuki/gocover-cobertura
```
Comment (zdw) [inline Makefile:45]: These are included in the testing container image, so can be omittted

**Comment by ray-milkey** [comment]
retest this please

File: Makefile (hunk 1 above)
Comment (ray-milkey) [reply Makefile]: golangci-lint installation is needed for developers

**Review by zdw — APPROVED** [review]
LGTM

**Pull request merged**